### ISSUE TITLE
Debugger with Python Adapter dowesn't work properly

### ISSUE THREAD

**Issue by Sawrz** [daveleroy/sublime_debugger#81]
Hi,
I have a strange issue regarding the debugger with the Python adapter, but let me give you some necessary information first:

I try to "debug" the following python code:

```
x = 5
some_list = []

for i in range(10):
    y = i + x
    z = i * x
    some_list.append(i)

a = sum(some_list)


print(a)
```
I set the breakpoint at line 4 (where the for-loop is). 

My projects configuration file:
```
{
	"build_systems":
	[
		{
			"file_regex": "^[ ]*File \"(...*?)\", line ([0-9]*)",
			"name": "Anaconda Python Builder",
			"selector": "source.python",
			"shell_cmd": "\"python3\" -u \"$file\""
		}
	],
	"folders":
	[
		{
			"path": "."
		}
	],
	"settings":
	{
		"debug.configurations":
		[
			{
				"request": "launch",
				"name": "Python: Current File",
				"type": "python",
				"program": "${file}",
				"debugOptions" : {
					"RedirectOutput": true,
				}
			},
		]
	}
}
```

I followed all steps in README.md:
1. Install Adapter
2. Create a Project in Sublime Text
3. Add Config
4. Open the debugger
5. Select Config
6. Set Breakpoint
7. Start Debugger

By starting the debugger, the sublime console gives me the following output:
```
Starting process: ['node', '/home/sandro/.config/sublime-text-3/Packages/Debugger/data/debug_adapters/vscode-python/extension/out/client/debugger/debugAdapter/main.js']
debug adapter process ended.
```

However, the debugger neither stops at the breakpoint nor stops at all. The debugger console shows the following information:
```
  code: 'MODULE_NOT_FOUND'
  requireStack: []
}

> click to input debugger command
```

But, if I click the pause button or any which is greyed out (the arrows), the sublime console gives me the following error:
```
Starting process: ['node', '/home/sandro/.config/sublime-text-3/Packages/Debugger/data/debug_adapters/vscode-python/extension/out/client/debugger/debugAdapter/main.js']
debug adapter process ended.
Traceback (most recent call last):
  File "/home/sandro/.config/sublime-text-3/Packages/Debugger/modules/core/core.py", line 48, in done
    raise exception
  File "/home/sandro/.config/sublime-text-3/Packages/Debugger/modules/libs/asyncio/tasks.py", line 239, in _step
    result = coro.send(None)
  File "/home/sandro/.config/sublime-text-3/Packages/Debugger/modules/debugger/debugger_session.py", line 390, in pause
    yield from self.client.Pause(self.command_thread)
  File "/home/sandro/.config/sublime-text-3/Packages/Debugger/modules/debugger/debugger_session.py", line 382, in command_thread
    return self.callstack.command_thread()
  File "/home/sandro/.config/sublime-text-3/Packages/Debugger/modules/debugger/debugger_session.py", line 577, in command_thread
    raise core.Error("No threads to run command")
Debugger.modules.core.error.Error: No threads to run command
```

Still, the debugger is running, and by clicking more greyed out buttons, I get the same error messages again.

I didn't try the same adapter in Visual Studio Code, but I successfully debugged the same code in VSC with the same settings (except for `"RedirectOutput"`) and the Python adapter from their marketplace.

Any idea where the problem is?

Best,
Sandro

**Comment by daveleroy** [comment]
This should be fixed in 0.2.8 this updates the adapter to use vscode python's new debugpy adapter.

I believe you will get an error about `debugOptions` not being a list so you will need to update your configuration to something like the following.

`"debugOptions":  ["RedirectOutput"]`

**Comment by Sawrz** [comment]
Thank you so much! 
Works now. So, I close this issue =)
